earth_enterprise/src/third_party/gdal/SConscript

```diff
@@ -145,6 +162,9 @@ openjpeg_extract = gdal_env.Command(
 #             kakadu_version, kakadu_version, kakadu_version, gdal_target))])
 
 
+# [8] 
```

Comment:
Probably ought to remove this since it's not used.

Reply:
yes

Reply:
done